conference.js

```diff
@@ -2024,6 +2034,77 @@ export default {
                             eventData
                         });
                     }
+
+                    if (eventData.name === CAPTURE_SCREENSHOT_MESSAGE) {
```

Comment:
Can we please add all this logic to a middleware instead? conference.js is legacy and we don't want to enmlarge any more if we can help it.